composables/useCountdown.ts

```diff
@@ -44,9 +42,9 @@ export default function useCountdown() {
 	}
 	
 	return {
-		displayDaysRemaining,
-		displayRemainingHours,
-		displayRemainingMinutes,
-		displayRemainingSeconds
```

Comment:
Suggestions for function API design consideration:

`useCountdown` is returning an object composed of individual functions at this point. 

if you console log it, you'll see:
```Object {
  "displayDaysRemaining": [Function displayDaysRemaining],
  "displayRemainingHours": [Function displayRemainingHours],
  "displayRemainingMinutes": [Function displayRemainingMinutes],
  "displayRemainingSeconds": [Function displayRemainingSeconds],
}
```

to grab the resultant value of days remaining, a dev will have to invoke the function: 

```
const currentDate = newDate();
const remainingDays = useCountdown(currentDate).displayDaysRemaining();

OR even with destructuring:

const {displayDaysRemaining} = useCountdown(currentDate);
const remainingDays = displayDaysRemaining();

```

I changed this up to `days()` so it's shorter, but it would be a DX improvement to assign any of those functions to a variable, and return that instead, so that a user can instantly grab the resultant value, like so:

`const {days} = useCountdown()` to mean remaining days

Reply:
great points on the logging and DX of this!

I think I was hesitant to rewrite the logic when going through each spec, more concerenced on meeting all of the test requirements 


Seeing these improvements, I can see I missed the most integral part of TDD which was influence design -i stopped at functional 🐿

greatly appreciate the in depth feedback